File: exercism/rna_transcription.py
def to_rna(dna_strand):
    def transcribe(nuc):
        if nuc == "C":
            return "G"
        elif nuc == "G":
            return "C"
        elif nuc == "A":
            return "U"
        else:
            return "A"

    return "".join(transcribe(nuc) for nuc in dna_strand)

File: exercism/rna-transcription.js
export const toRna = dna => {
    return [...dna].map(nuc => {
        switch (nuc) {
          case 'G':
            return 'C';
          case 'C':
            return 'G';
          case 'T':
            return 'A';
          case 'A':
            return 'U';
        }
    }).join("");
};
